src/data/constants.ts

```diff
@@ -13,3 +13,5 @@ export const kPinColorAttributeName = "pinColor";
 export const kMapName = "NeoMap";
 
 export const kSliderComponentName = "NeoSlider";
+
+export const kOneMonthInSeconds = 259200;
```

Comment:
This looks like it is missing a zero.
60*60*24*30 = 2,592,000

I'd recommend just putting the math into the code to make it clear:
```suggestion
export const kOneMonthInSeconds = 60*60*24*30;
```
In the future we might want to make this dynamic based on which month it is so it correctly covers Feb. and the months with 31 day, but this seems fine for now.